cockpit/src/subscriptions-register.jsx

```diff
@@ -65,6 +67,24 @@ class PatternDialogBody extends React.Component {
                 </form>
             );
         }
+        let insights;
+        if (this.props.insights_available) {
+            insights = [
+                <label key="0" className="control-label" htmlFor="subscription-insights">
+                    {_("Insights")}
+                </label>,
+                <label key="1" className="checkbox-inline">
+                    <input id="subscription-insights" type="checkbox" checked={this.props.insights}
+                           onChange={value => this.props.onChange('insights', value)}/>
+                        {_("Connect this system to Red Hat Insights.")}
```

Comment:
The **Red Hat Insights** parts needs a link to https://www.redhat.com/en/technologies/management/insights so that people can read more about it if they are not familiar enough with it.
This is the same url that's on the [RHEL benefits and features page](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux#features)

Reply:
Done. I have also removed the (i) tooltip thingy.